packages/magika.vm/magika.vm.nuspec

```diff
@@ -0,0 +1,13 @@
+<?xml version="1.0" encoding="utf-8"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2015/06/nuspec.xsd">
+  <metadata>
+    <id>magika.vm</id>
+    <version>0.5.0</version>
+    <authors>Google</authors>
```

Comment:
According to the [project page](https://github.com/google/magika?tab=readme-ov-file#citation), this is not an official Google project and the authors are: `Fratantonio, Yanick and Invernizzi, Luca and Zhang, Marina and Metitieri, Giancarlo and Kurt, Thomas and Galilee, Francois and Petit-Bianco, Alexandre and Farah, Loua and Albertini, Ange and Bursztein, Elie`